engineio/Cargo.toml

```diff
@@ -29,7 +29,8 @@ async-trait = "0.1.77"
 async-stream = "0.3.5"
 thiserror = "1.0"
 native-tls = "0.2.11"
-url = "2.4.1"
+url = "2.5.0"
+rmp = "0.8"
```

Comment:
Can we hide this serializer feature behind a feature flag? I don't think we need this everywhere, but rather have it enabled if necessary. For general work with feature flags, you can take a look here: https://doc.rust-lang.org/cargo/reference/features.html.

Reply:
I have remove this part of code and left only custom serializer support